node/src/sub_lib/wallet.rs

```diff
@@ -327,73 +325,63 @@ impl<'de> serde::Deserialize<'de> for Wallet {
-                    serde::__private::Some(address) => address,
-                    serde::__private::None => {
-                        return serde::__private::Err(serde::de::Error::invalid_length(
+                    Some(address) => address,
+                    None => {
+                        return Err(serde::de::Error::invalid_length(
                             0usize,
                             &"struct Wallet with 1 element",
                         ));
                     }
                 };
-                serde::__private::Ok(Wallet {
+                Ok(Wallet {
                     kind: WalletKind::Address(address),
                 })
             }
             #[inline]
-            fn visit_map<A>(self, mut map: A) -> serde::__private::Result<Self::Value, A::Error>
+            fn visit_map<A>(self, mut map: A) -> Result<Self::Value, A::Error>
             where
                 A: serde::de::MapAccess<'de>,
             {
-                let mut possible_address: serde::__private::Option<Address> =
-                    serde::__private::None;
-                while let serde::__private::Some(key) =
-                    serde::de::MapAccess::next_key::<WalletField>(&mut map)?
-                {
+                let mut possible_address: Option<Address> = None;
+                while let Some(key) = serde::de::MapAccess::next_key::<WalletField>(&mut map)? {
                     match key {
                         WalletField::Address => {
-                            if serde::__private::Option::is_some(&possible_address) {
-                                return serde::__private::Err(
-                                    <A::Error as serde::de::Error>::duplicate_field("address"),
-                                );
+                            if Option::is_some(&possible_address) {
+                                return Err(<A::Error as de::Error>::duplicate_field("address"));
                             }
                             possible_address = match &self.human_readable {
-                                true => serde::__private::Some(serde::de::MapAccess::next_value::<
-                                    Address,
-                                >(
-                                    &mut map
-                                )?),
+                                true => {
+                                    Some(serde::de::MapAccess::next_value::<Address>(&mut map)?)
+                                }
                                 false => {
                                     let bytes =
                                         serde::de::MapAccess::next_value::<Vec<u8>>(&mut map)?;
                                     let mut address = [0u8; 20];
                                     address.copy_from_slice(bytes.as_slice());
-                                    serde::__private::Some(H160(address))
+                                    Some(Address { 0: address })
```

Comment:
@FinsaasGH: Clippy doesn't like this. According to the Linux build in Actions, it wants `Address (address)`.